pkg/assets/currencies.go

```diff
@@ -0,0 +1,191 @@
+package assets
+
```

Comment:
nit: should we rename this file to link?
Not directly related to this, but we tend to create large files that contains code for multiple types. Is not it easier to navigate packages with large number of small files?